server/Dockerfile

```diff
@@ -41,19 +41,18 @@ WORKDIR /usr/src/supply-chain
 
 # Install Dependencies
 COPY asset_client/package.json asset_client/
-RUN cd asset_client/ && npm install && cd ..
-
 COPY fish_client/package.json fish_client/
-RUN cd fish_client/ && npm install && cd ..
-
 COPY server/package.json server/
-RUN cd server/ && npm install && cd ..
+
+RUN cd asset_client/ && npm install && cd .. \
+ && cd fish_client/ && npm install && cd .. \
+ && cd server/ && npm install && cd ..
```

Comment:
My understanding is that because of how Docker cache's each step, it is better to separate out dependency installs into pairs of `COPY` + `RUN`. This way if a change was made to the server dependencies, you wouldn't have to go back and reinstall the client dependencies again.

With all of the installs in one command, I believe you will have to reinstall client dependencies every time a server dependency changes.